eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -92,6 +99,8 @@ function Test-Exit0WhenAllFilesExcluded() {
     # Act
     &"$PSScriptRoot/check-spelling-in-changed-files.ps1" `
         -TargetBranch HEAD~1 `
+        -CspellConfigPath "./.vscode/cspell.json" `
```

Comment:
Isn't this already defaulted?

Reply:
These tests are executed in the scope of a temporary folder. The default value used in the parameters is set to a location which is relative to `check-spelling-in-changed-files.ps1` so it will always point to the cspell.json file for the repo. 

One benefit of using the default value for `CspellConfigPath` is that one can run `check-spelling-in-changed-files.ps1` from any location inside the repo and still get useful results.